glossary.yml

```diff
@@ -10404,6 +10404,12 @@
       In [deep learning](#deep_learning), an epoch is one cycle in the deep learning process where all
       the training data has been fed to the algorithm once. Training a deep neural networks usually
       consists of multiple epochs.
+  es:
+    term: epoch (deep learning)
+    def: > 
+      En [deep learning](#deep_learning), un epoch es un ciclo completo en el proceso del aprendizaje profundo 
```

Comment:
Here we are using "deep learning" and "aprendizaje profundo" at the same time. Should we keep just "aprendizaje profundo"?